doc/src/learn/architecture/consensus.md

```diff
@@ -2,17 +2,19 @@
 title: Narwhal and Tusk, Sui's Consensus Engine
 ---
 
-This is a brief introduction to [Narwhal and Tusk](https://github.com/MystenLabs/narwhal), the high-throughput mempool and consensus offered by Mysten Labs. Sui runs consensus as needed to periodically checkpoint its state. And for those transactions that require a total ordering, Narwhal/Tusk is the consensus engine of Sui.
+This is a brief introduction to [Narwhal and Bullshark](https://github.com/MystenLabs/narwhal), the high-throughput mempool and consensus offered by Mysten Labs. Sui runs consensus as needed to periodically checkpoint its state. And for those transactions that require a total ordering, Narwhal/Bullshark is the consensus engine of Sui.
 
 The dual name highlights that the systems split the responsibilities of:
-- ensuring the availability of data submitted to consensus (Narwhal)
-- agreeing on a specific ordering of this data (Tusk)
+- ensuring the availability of data submitted to consensus ([Narwhal](https://arxiv.org/abs/2105.11827))
+- agreeing on a specific ordering of this data ([Bullshark](https://arxiv.org/abs/2201.05677))
+
+> **Note:** Bullshark has replaced the Tusk component of the consensus protocol for reduced latency and support for fairness (where even slow validators can contribute). See [DAG Meets BFT - The Next Generation of BFT Consensus](https://decentralizedthoughts.github.io/2022-06-28-DAG-meets-BFT/) for a comparison of the protocols.
 
 This is done in two layered modules, so Narwhal can also be used coupled with an external consensus algorithm, such as HotStuff, Istanbul BFT, or Tendermint. Narwhal is undergoing integration in the [Celo](https://www.youtube.com/watch?v=Lwheo3jhAZM) and [Sommelier](https://www.prnewswire.com/news-releases/sommelier-partners-with-mysten-labs-to-make-the-cosmos-blockchain-the-fastest-on-the-planet-301381122.html) blockchain.
 
-Narwhal and Tusk represent the latest variant of decades of work on multi-proposer, high-throughput consensus algorithms that reaches throughputs more than 130,000 transactions per second on a WAN, with production cryptography, permanent storage, and a scaled-out primary-worker architecture.
+Narwhal and Bullshark represent the latest variant of decades of work on multi-proposer, high-throughput consensus algorithms that reaches throughputs more than 130,000 transactions per second on a WAN, with production cryptography, permanent storage, and a scaled-out primary-worker architecture.
```

Comment:
The Bullshark paper mentions in the abstract that it achieves "125,000 transaction per second
with a 2 seconds latency for a deployment of 50 parties"
cc @huitseeker

Reply:
Thank you! I've added this.